Quantum-Control-Applications/Superconducting/Single Flux Tunable Transmon/SSB mixer for readout/IQ_blobs.py

```diff
@@ -76,7 +76,7 @@
 #####################################
```

Comment:
This IQ blobs code uses active reset, but that means that it cannot be run as is. (Chicken and egg)...
Is there any reason why it is different from the fixed qubit code?

Reply:
During the installations I made, they were really interested in seeing active reset, even if they don't have single shot (they just want to see how we can move one IQ blob to the other) so the idea is to first comment the active reset part to rotate the blobs and then show the active reset.
Maybe I can split it into two files or indeed use the one in single fixed transmon

Reply:
Yes, I agree. If you take a look at the fixed transmon case, you can see that all of the active reset parts are commented out at the end of the file.

I also wondered what is the best, maybe a separate file, maybe not... But in any case, I think that we should supply "ready to run files", and files that can be understood even without us present.
"Files for dummies :)"

Reply:
So I created a macros that performs active reset a given number of times until success and created a file that measures afterward to show the IQ blobs after active reset, all comments are welcome :)

Reply:
Cool, looks good.
BTW, in the original script, because there was the cooldown inside the macro, then it wasn't "really" with active reset...

Reply:
Yes I realized it :)